custom/ewsghana/reports/email_reports.py

```diff
@@ -0,0 +1,222 @@
+from casexml.apps.stock.models import StockTransaction
+from corehq.apps.commtrack.models import StockState, CommtrackConfig
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.reports.commtrack.const import STOCK_SECTION_TYPE
+from corehq.apps.reports.datatables import DataTablesHeader, DataTablesColumn
+from corehq.apps.reports.filters.fixtures import AsyncLocationFilter
+from corehq.apps.reports.filters.dates import DatespanFilter
+from custom.ewsghana.reports import EWSData, MultiReport
+from django.utils.translation import ugettext as _
+from custom.ewsghana.utils import get_supply_points
+from dimagi.utils.decorators.memoized import memoized
+
+
+def get_stock_summary_locations(loc_id, domain):
+    return get_supply_points(loc_id, domain)
```

Comment:
is there a reason this method is here? in case there's some future extension or just for readabilitiy?